koku/api/tags/gcp/openshift/queries.py

```diff
@@ -27,12 +26,16 @@ class OCPGCPTagQueryHandler(GCPTagQueryHandler, OCPTagQueryHandler):
         {
             "db_table": OCPGCPTagsSummary,
             "db_column_period": "cost_entry_bill__billing_period",
-            "annotations": {"enabled": Exists(GCPEnabledTagKeys.objects.filter(key=OuterRef("key")))},
+            "annotations": {
+                "enabled": Exists(EnabledTagKeys.objects.filter(provider_type="GCP").filter(key=OuterRef("key")))
```

Comment:
This is probably fine, but everywhere else, we use `provider_type=provider` where `provider = Provider.provider-type`. So maybe here we can use `provider_type=Provider.PROVIDER_GCP`?
```suggestion
                "enabled": Exists(EnabledTagKeys.objects.filter(provider_type=Provider.PROVIDER_GCP).filter(key=OuterRef("key")))
```
and same for Provider.PROVIDER_OCP below?